examples/with-redux-saga/saga.js

```diff
@@ -1,6 +1,6 @@
 /* global fetch */
 
-import { delay } from 'redux-saga'
+import { delay } from 'redux-saga/effects'
 import { all, call, put, take, takeLatest } from 'redux-saga/effects'
```

Comment:
```suggestion
import { all, call, put, take, takeLatest, delay } from 'redux-saga/effects'
```

Reply:
Seems like I've been a bit too tired yesterday evening. There you go: https://github.com/zeit/next.js/pull/6109/commits/6f7191dc1c2db81982b331a01943201a0abd7eb4.